test/unit_tests/modules/FlyoutMenu-spec.js

```diff
@@ -0,0 +1,449 @@
+      var args = expandBeginSpy.getCall( 0 ).args[0];
+      expect( args.target ).to.equal( flyoutMenu );
+      expect( args.type ).to.equal( 'expandBegin' );
+
+      expect( expandEndSpy.callCount ).to.equal( 1 );
+      args = expandEndSpy.getCall( 0 ).args[0];
+      expect( args.target ).to.equal( flyoutMenu );
+      expect( args.type ).to.equal( 'expandEnd' );
+
+      // Check expected aria-expanded state.
+      expect( triggerDom.getAttribute( 'aria-expanded' ) ).to.equal( 'true' );
+      expect( contentDom.getAttribute( 'aria-expanded' ) ).to.equal( 'true' );
+      expect( altTriggerDom.getAttribute( 'aria-expanded' ) )
+        .to.equal( 'true' );
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called by trigger click', function() {
+      triggerDom.click();
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called by alt trigger click', function() {
+      altTriggerDom.click();
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called directly', function() {
+      flyoutMenu.expand();
+    } );
+  } );
+
+  describe( '.collapse()', function() {
+    beforeEach( function() {
+      // Set up expected event listeners.
+      collapseBeginSpy = sinon.spy();
+      collapseEndSpy = sinon.spy();
+
+      flyoutMenu.init();
+      flyoutMenu.addEventListener( 'collapseBegin', collapseBeginSpy );
+      flyoutMenu.addEventListener( 'collapseEnd', collapseEndSpy );
+      triggerDom.click();
+    } );
+
+    afterEach( function() {
+      // Check expected event broadcasts.
+      expect( collapseBeginSpy.callCount ).to.equal( 1 );
+      var args = collapseBeginSpy.getCall( 0 ).args[0];
+      expect( args.target ).to.equal( flyoutMenu );
+      expect( args.type ).to.equal( 'collapseBegin' );
+
+      expect( collapseEndSpy.callCount ).to.equal( 1 );
+      args = collapseEndSpy.getCall( 0 ).args[0];
+      expect( args.target ).to.equal( flyoutMenu );
+      expect( args.type ).to.equal( 'collapseEnd' );
+
+      // Check expected aria-expanded state.
+      expect( triggerDom.getAttribute( 'aria-expanded' ) ).to.equal( 'false' );
+      expect( contentDom.getAttribute( 'aria-expanded' ) ).to.equal( 'false' );
+      expect( altTriggerDom.getAttribute( 'aria-expanded' ) )
+        .to.equal( 'false' );
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called by trigger click', function() {
+      triggerDom.click();
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called by alt trigger click', function() {
+      altTriggerDom.click();
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called directly', function() {
+      flyoutMenu.collapse();
+    } );
+  } );
+
+  describe( '.setExpandTransition()', function() {
+    xit( 'should set a transition', function( done ) {
```

Comment:
Whoops, this shouldn't be xit'd! Fixed in https://github.com/cfpb/cfgov-refresh/pull/1724/commits/cefffd4121f2de9e9c21498edb3c87927c28594e